### DISCUSSION TITLE
Disable auto injecting markdown links when copy-pasting links

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#78894] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

THis is really annying. I write my markdown by hand and each time I try to paste the URL inside markdown links: `[something](|)` GitHub injects full markdown links inside my markdown links. I need to copy the URL into my editor (desktop one) and copy it again in order to paste the URL.

Can there be an option to disable any autoformatting (except maybe those on enter, like lists) into the markdown editor? Or maybe you can just detect that I'm already inside the link and don't put links inside my links.

**Comment by imcotton** [comment]
There is a **URL paste behavior** setting under **Accessibility** in user preferences. I have specifically set it to **Plain text**, but it does not seem to be taking effect. Is this feature currently broken?
